APIcall/data_collection.py

```diff
@@ -0,0 +1,41 @@
+import pandas as pd
+import requests
+import argparse
+from datetime import date
+
+def api_311(start_date, end_date, skip, limit):
+    skip = 0
+    limit = 10000
+    data_year = []

```

Comment:
nit: `data_year` is a bit misleading; maybe `all_requests`?

Reply:
Thanks! Changed the variable name.